### PR TITLE
Mayhem Repo Integration

### PR DESCRIPTION
**Pull request by rishabh1024**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:20.04
```
Comment (xansec) [inline Dockerfile]: Please add a package stage to reduce image size here.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+image: ghcr.io/rishabh1024/drlibs-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: You may omit the image tag as this will be populated during runtime.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Dockerfile (hunk 1 above)
Comment (rishabh1024) [reply Dockerfile]: I have made the changes

File: Mayhemfile (hunk 2 above)
Comment (rishabh1024) [reply Mayhemfile]: I have made the changes

**Pull request merged**